include/partition/cell_storage.hpp

```diff
@@ -117,33 +123,27 @@ template <bool UseShareMemory> class CellStorageImpl
             std::size_t stride;
         };
 
-        std::size_t GetRow(NodeID node) const
```

Comment:
What is the idea here behind inlining these functions below and removing the interface?